app/src/main/java/com/elavon/converge/model/type/ElavonTransactionType.java

```diff
@@ -32,7 +32,7 @@
     PINLESS_DEBIT_SALE("pldpurchase"),
     CASH_SALE("cashsale"),
     CASH_CREDIT("cashcredit"),
-    EMV_CT_SALE("emvchipsale"),
+    EMV_CT_SALE("EMVCHIPSALE"),
```

Comment:
only uppercase works?

Reply:
good point - let me check - it was my desperate attempt to see if that was causing the failure :-)